### DISCUSSION TITLE
Freeze column - What for?

### DISCUSSION THREAD

**Discussion by Vadorequest** [supabase/supabase#621] [Questions] [answered]
I noticed we have the ability to freeze a column, but couldn't find documentation about what are the use-cases for doing so.

![image](https://user-images.githubusercontent.com/3807458/106037432-f5522000-60d6-11eb-9b58-d3385ab36df8.png)

It seems the goal is to avoid a column to be edited, but when do people actually use this in real life?

**Comment by kiwicopple** [comment, marked as answer]
Hey @Vadorequest ! This is for very wide tables where you want to "fix" the column to the left:


https://user-images.githubusercontent.com/10214025/106132813-61985680-619f-11eb-8c2c-45b67f0a6b10.mov


This is just a UI helper - it won't affect your database in any way 👍
